cloud-infrastructure/storage/block-storage/README.md

```diff
@@ -0,0 +1,84 @@
+# Block Storage
+
+[High-performance block storage at any scale](https://www.oracle.com/cloud/storage/#rc30p1)
+
+We've designed our storage platform as an ideal complement to Oracle compute and networking services to support the highest performance requirements. OCI Block Volume uses the latest NVMe SSDs and provides nonblocking network connectivity to every host. Oracle delivers consistent, low-latency performance of up to 225 IOPS/GB to a maximum of 300,000 IOPS and 2,680 MB/sec of throughput per volume.
+
+Features
+
+- Cross-region asynchronous block volume replication
+- Consistent high performance
+- Dynamic performance scaling with auto-tuning of block volumes
+- Scales volumes from 50 GB to 32 TB
+- Attach volumes to reach 1 PB per compute instance
+- Less than half the price of AWS EBS
```

Comment:
Please remove direct reference to competition.

Reply:
done